src/Microsoft.Azure.SignalR.AspNet/EndpointProvider/ServiceEndpointProvider.cs

```diff
@@ -46,8 +47,13 @@ public ServiceEndpointProvider(ServiceEndpoint endpoint, ServiceOptions options)
             _port = endpoint.Port;
             _algorithm = options.AccessTokenAlgorithm;
             Proxy = options.Proxy;
-        }
 
+            if (!_accessKey.InitializedTask.IsCompleted)
+            {
+                var app = AADHelper.BuildApplication(options.AzureAdOptions);
```

Comment:
`AAD`=> let's stick to one naming convention for aad all the way through: `AzureActiveDirectory` or `Aad`